<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyWorkoutsScreen</class>
 <widget class="QWidget" name="MyWorkoutsScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[objectName|=&quot;invisibleButton&quot;]{border: none; }
QWidget{font-family: Tamworth Gothic}</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/My Workouts.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="invisibleButton_6">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>180</y>
     <width>451</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="downArrowLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>690</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/My Workouts/images/Down Arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="upArrowLabel">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>280</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/My Workouts/images/Up-Arrow.png</pixmap>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>330</y>
     <width>456</width>
     <height>351</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background: black;
border: none;</string>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>456</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_upArrowButton" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>270</y>
     <width>108</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
  <widget class="InvisibleButton" name="invisibleButton_downArrowButton" native="true">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>680</y>
     <width>108</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text" stdset="0">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InvisibleButton</class>
   <extends>QWidget</extends>
   <header>invisiblebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
